taxonomy/algolia/serializers.py

```diff
@@ -132,6 +135,19 @@ def get_similar_jobs(self, obj):
         recommendations_data = self.context.get('jobs_with_recommendations', None)
         return self.extract_similar_jobs(recommendations_data, obj.name)
 
+    def get_b2c_opt_in(self, obj):
+        """
+        Checks to see if the job is present on our job allowlist. This attribute will be primarily used to filter the
+        list of available jobs a learner can select from in the B2C Skills Builder.
+
+        Arguments:
+            obj (Job): Job instance, used so we can compare its external id to the items on the allowlist
+        """
+        if hasattr(settings, "TAXONOMY_B2C_JOB_ALLOWLIST"):
+            allowlist = settings.TAXONOMY_B2C_JOB_ALLOWLIST
```

Comment:
How big is this list?

Reply:
@irfanuddinahmad IIRC, it isn't very large -- well under a hundred, but closer to 50.

Reply:
can we use django admin to define the allow list

Reply:
@irfanuddinahmad By introducing a new model? Or is there another mechanism I can use to manage it via django admin?

I was really hoping to _not_ introduce a new model in case this isn't a solution we stick with long term. The setting (and configuration change) is easier to remove if we decide this isn't working for us.

cc: @hurtstotouchfire

Reply:
@justinhynes Would the BlockedList be smaller than the AllowList? ... is the AllowList almost equal to the total number of jobs in the system?

Reply:
@justinhynes I was thinking that it would make this a lot simpler if instead of introducing this new field `b2c_opt_in` on the backend side, we introduce the `TAXONOMY_B2C_JOB_ALLOWLIST` as a constant on the frontend side and do the filtering there?

If frontend is directly accessing the API and we have control over the frontend code then this makes more sense?

Reply:
@irfanuddinahmad Yes, the blocklist would be much much larger. 

@saleem-latif -- It would be much simpler if there was an attribute in the index that we can filter on so we could continue to use algolia instant search. I don't think it would be ideal to do the processing on the frontend. The frontend isn't accessing the REST API's of the taxonomy connector at all -- we're relying on Algolia for the search experience.

Reply:
@hurtstotouchfire @justinhynes, Please go ahead if this is a temporary change that will be converted to a more permanent solution or reverted.